cli/smartcontract/smart_contract.go

```diff
@@ -761,7 +767,7 @@ func contractDeploy(ctx *cli.Context) error {
 		return err
 	}
 
-	txScript, err := request.CreateDeploymentScript(nefFile.Script, m)
+	txScript, err := request.CreateDeploymentScript(f, m)
```

Comment:
Actually, it can be made accept Nef and Manifest to ensure that it's not random bytes we're feeding it with. It's not on the hot path, so I think it's not an issue, we better use safer interface (but we also need to reuse it in `NewDeployTx`).